spec/integration/review_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'spec_helper'
+
+describe 'project show page with reviews' do
+  it 'shows all reviews in a project' do
+    project = create(:project, title: "Java Tic-Tac-Toe")
+    review = create(:review, content: "Looks good!", project_id: project.id)
+
+    visit "/reviews"
```

Comment:
I think that reviews should live on the project show page like so:

visit '/projects/' + project.id.to_s